wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/TaskDefinitionElement.scala

```diff
@@ -1,5 +1,7 @@
 package wdl.model.draft3.elements
 
-case class TaskDefinitionElement(identifier: String
-                                // TODO: sections
+case class TaskDefinitionElement(name: String,
+                                inputsSection: Option[InputsSectionElement],
+                                outputsSection: Option[OutputsSectionElement],
+                                runtimeSection: Option[RuntimeSectionElement]
```

Comment:
Is `command: CommandSectionElement` still in the works?

Reply:
Yes, wanted to start with a positive task definition conversion before adding new conversions.